firebase-functions/functions/src/choiceOperations.ts

```diff
@@ -4,19 +4,28 @@ export function hasEveryoneChosen(players: Array<FirebaseFirestore.QueryDocument
   return players.every(player => (player.data() as Player).choice);
 }
 
-export function getLosers(players: Array<FirebaseFirestore.QueryDocumentSnapshot<FirebaseFirestore.DocumentData>>): string[] {
-  if (isDraw(players)) {
-    return [];
-  }
-  // this is just to show something. soon, all the losers will be calculated and returned here instead
-  return [players[0].id];
+export function getActiveLosers(players: Array<FirebaseFirestore.QueryDocumentSnapshot<FirebaseFirestore.DocumentData>>): FirebaseFirestore.QueryDocumentSnapshot<FirebaseFirestore.DocumentData>[] {
+  return isDraw(players)
```

Comment:
I think we should do the filtering based on `isObserver` before, because it part now as I see now to determine whether is draw or not and even of getting the losers, or I am missing a piece?

Reply:
I am just stupid I think. I _am_ doing the filtering before, I think it can just be removed here

Reply:
Done